cachelib/allocator/CacheAllocator.h

```diff
@@ -710,6 +711,16 @@ class CacheAllocator : public CacheBase {
   // @return    the full usable size for this item
   uint32_t getUsableSize(const Item& item) const;
 
+  // create memory assignment to bg workers
+  auto createBgWorkerMemoryAssignments(size_t numWorkers);
+
+  // whether bg worker should be woken
+  bool shouldWakeupBgEvictor(PoolId pid, ClassId cid);
+
+  // bg workers responsible for promoting/evicting from particlar pool and
+  // allocation calss
+  size_t backgroundWorkerId(PoolId pid, ClassId cid, size_t numWorkers);
```

Comment:
can this be a static method in BackgroundMover instead?

Reply:
Sure, done.